CDTDatastore/HTTP/CDTSessionCookieInterceptorBase.m

```diff
@@ -78,6 +78,22 @@ - (CDTHTTPInterceptorContext *)interceptResponseInContext:(CDTHTTPInterceptorCon
         // Clear the cookies as we are no longer authorized
         _cookies = nil;
         context.shouldRetry = YES;
+    } else if (context.response.statusCode == 403) {
+        NSError *error = nil;
+        NSDictionary *statusCodeMessage = [NSJSONSerialization JSONObjectWithData:context.responseData
+                                                                        options:0
+                                                                          error:&error];
+        if(!error) {
+            NSString *http403Error = [[NSString alloc] init];
+            http403Error = [statusCodeMessage objectForKey:@"error"];
+            if([http403Error containsString:@"credentials_expired"]) {
```

Comment:
We can probably use `isEqualTo` here as we want to match the exact string

Reply:
Fixed in 9b2f6bb.